ui/app/modifiers/keyboard-shortcut.js

```diff
@@ -1,44 +1,31 @@
 import { inject as service } from '@ember/service';
 import Modifier from 'ember-modifier';
 import { registerDestructor } from '@ember/destroyable';
-import { assert } from '@ember/debug';
 
 export default class KeyboardShortcutModifier extends Modifier {
   @service keyboard;
+  @service router;
 
-  /**
-   * For Dynamic/iterative keyboard shortcuts, our patterns look like "Shift+0" by default
-   * Do a couple things to make them more human-friendly:
-   * 1. Make them 1-based, instead of 0-based
-   * 2. Prefix numbers 1-9 with "0" to make it so "Shift+10" doesn't trigger "Shift+1" then "0", etc.
-   * ^--- stops being a good solution with 100+ row lists/tables, but a better UX than waiting for shift key-up otherwise
-   *
-   * @param {string[]} pattern
-   */
-  cleanPattern(pattern) {
-    let patternNumber = pattern.length === 1 && pattern[0].match(/\d+/g);
-    if (!patternNumber) {
-      return pattern;
-    } else {
-      patternNumber = +patternNumber[0]; // regex'd string[0] to num
-      patternNumber = patternNumber + 1; // first item should be Shift+1, not Shift+0
-      assert(
-        'Dynamic keyboard shortcuts only work up to 99 digits',
-        patternNumber < 100
-      );
-      pattern = [`Shift+${('0' + patternNumber).slice(-2)}`]; // Shift+01, not Shift+1
+  modify(
+    element,
+    _positional,
+    {
+      label,
+      pattern = '',
+      action = () => {},
+      menuLevel = false,
+      enumerated = false,
```

Comment:
In this PR that's true, but the `keyboard-shortcut` modifier is used elsewhere with enumerated false in many places (main nav for example)

Since it adds behaviour, I'd prefer to have the default state default to false here